<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1065</width>
    <height>599</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>20</pointsize>
    <underline>true</underline>
    <strikeout>false</strikeout>
    <kerning>true</kerning>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="PolygonWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>671</width>
      <height>531</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>10</y>
      <width>361</width>
      <height>521</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout" rowstretch="0,0,0,0,0,0,0,0,0">
     <property name="rightMargin">
      <number>0</number>
     </property>
     <item row="0" column="1">
      <widget class="QRadioButton" name="isSpline">
       <property name="text">
        <string>Произвольная кривая</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0" colspan="2">
      <widget class="EdgeTableWidget" name="listWidget"/>
     </item>
     <item row="0" column="0">
      <widget class="QRadioButton" name="isPolygon">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>Многоугольник</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QPushButton" name="lineColorButton">
       <property name="text">
        <string>Цвет линии</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QPushButton" name="colorButton">
       <property name="text">
        <string>Цвет закраски</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0" colspan="2">
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>Выбрать затравочную точку</string>
       </property>
      </widget>
     </item>
     <item row="8" column="0" colspan="2">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Время последнего заполнения: </string>
       </property>
      </widget>
     </item>
     <item row="7" column="0" colspan="2">
      <widget class="QLabel" name="fillPoint">
       <property name="text">
        <string>Координаты затравочной точки: (0, 0)</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0" colspan="2">
      <widget class="QCheckBox" name="hasDelay">
       <property name="text">
        <string>Задержка</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1065</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PolygonWidget</class>
   <extends>QWidget</extends>
   <header>polygonwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>EdgeTableWidget</class>
   <extends>QListWidget</extends>
   <header>edgetablewidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
